src/main/java/Duke.java

```diff
@@ -1,10 +1,269 @@
+     */
+    public static void printNotCommand() {
+        System.out.println(
+                DIVIDER
+                        + " Sorry, your command is not recognized\n"
+                        + " See 'help'. \n"
+                        + DIVIDER_LINE_ONLY
+        );
+    }
+
+    /**
+     * Prints the help message.
+     */
+    public static void printHelp() {
+        String helpMessage = " 'help'     : Display tips on using this application\n"
+                + " 'exit'     : Exit the application\n"
+                + " 'bye'      : Exit the application\n"
+                + " 'list'     : List all type of tasks you added to your task list\n"
+                + " 'done'     : Mark a task as done\n\t"
+                + " e.g. 'done 2' will mark the second task as done\n"
+                + " 'todo'     : Add a new todo task\n\t"
+                + " e.g. 'todo read book' will add \"read book\" to your task list\n"
+                + " 'deadline' : Add a new deadline task\n\t"
+                + " e.g. 'deadline read book /by Sunday' will"
+                + " add \"read book\" to your task list with deadline \"Sunday\"\n"
+                + " 'event' : Add a new event task\n\t"
+                + " e.g. 'event read book /at Mon 2-4pm' will"
+                + " add \"read book\" to your task list with period  \"Mon 2-4pm\"\n";
+
+        System.out.println(
+                DIVIDER_LINE_ONLY + DIVIDER
+                        + helpMessage
+                        + DIVIDER_LINE_ONLY + DIVIDER_LINE_ONLY
+        );
+    }
+
+    /**
+     * Gets command from user input.
+     *
+     * @param userInput user's input.
+     * @return command.
+     */
+    private static String getCommand(String userInput) {
+        String[] inputWords = userInput.split(" ");
+        return inputWords[0];
+    }
+
+    /**
+     * Gets user input and execute corresponding command until the loop is exit.
+     *
+     * @param sc       Java Scanner to get user input.
+     * @param taskList The list of Task objects.
+     */
+    private static void loopOperation(Scanner sc, TaskList taskList) {
+        String userInput;
+        while (true) {
+            userInput = sc.nextLine();
+            String userCommand = getCommand(userInput);
+            switch (userCommand) {
+            case "help":
+                printHelp();
+                break;
+            case "list":
+                taskList.printCurrentList();
+                break;
+            case "todo":
+                addTodoTaskToList(taskList, userInput);
+                break;
+            case "deadline":
+                addDeadlineTaskToList(taskList, userInput);
+                break;
+            case "event":
+                addEventTaskToList(taskList, userInput);
+                break;
+            case "done":
+                handleDoneTask(taskList, userInput);
+                break;
+            case "exit":
```

Comment:
During fall-through cases, where there are no break statements, it is good practice to include a comment like //FALL-THROUGH.

Reply:
noted, thank you